_content/articles/natures-no-thingness_marek-sullivan.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Nature’s No-Thingness: Holistic Eco-Buddhism and the Problem of Universal Identity"
+authors:
+  - Marek Sullivan
+external_url: "https://blogs.dickinson.edu/buddhistethics/files/2015/07/JBE-Sullivan.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1Z6RScCQU0Dz-oBQqS7fdPcP6vWxQZxtI/view?usp=share_link"
+formats: [pdf]
+course: nature
+tags:
+  - environmentalism
+  - chinese buddhism
+  - huayan
+  - madyamaka
+  - pratityasamutpada
+year: 2015
+oclc: 644130897
```

Comment:
It's okay to include I guess, but oclc is a bit odd for an article when the ID is for the journal as a whole...